README.md

```diff
@@ -1,4 +1,4 @@
-# The Internal Structure of Pama-Nyungan
+# CLDF dataset derived from Bowern and Atkinson's "Internal Structure of Pama-Nyungan", from 2012
```

Comment:
No comma after the "TITLE" I think?

Reply:
I though we had decided for a comma after all? I put in in email suggestion, but it was easy to overlook it...

@LinguList , could you confirm?

Reply:
It's not needed.